src/control/cmd/daos/util.go

```diff
@@ -62,6 +63,25 @@ func apiVersion() string {
 	)
 }
 
+func srvBuildInfo() (*build.Info, error) {
+	var major uint32
+	var minor uint32
+	var patch uint32
+	var tagPtr *C.char
+
+	rc := C.dc_mgmt_srv_version((*C.uint)(&major), (*C.uint)(&minor), (*C.uint)(&patch), &tagPtr)
+	if err := daosError(rc); err != nil {
+		return nil, err
+	}
```

Comment:
I wonder if it would be better to do this 100% in Go, instead of having a C function.

Reply:
How would that work? Remember that this is the `daos` utility, which relies on libdaos to talk to the agent...